core/src/main/java/net/adoptopenjdk/icedteaweb/jnlp/element/application/AppletDesc.java

```diff
@@ -44,6 +44,7 @@
     public static final String NAME_ATTRIBUTE = "name";
     public static final String WIDTH_ATTRIBUTE = "width";
     public static final String HEIGHT_ATTRIBUTE = "height";
+    public static final String PROGRESS_CLASS_ATTRIBUTE = "progress-class";
```

Comment:
Is this used in the code?

Reply:
Not now, this PR is about implement proper parsing of progress-class attribute. Making use of it will be a separate PR.